client/src/game/ui/HanabiCardInit.ts

```diff
@@ -255,11 +257,16 @@ export function pips(variant: Variant) {
     const rankPip = new RankPip({
       x,
       y,
+      fontFamily: 'Arial',
+      fontStyle: 'bold',
+      fontSize: 63,
+      align: 'center',
+      text: rank.toString(),
       width: Math.floor(CARD_H * 0.1),
       height: Math.floor(CARD_H * 0.1),
       fill: 'black',
       stroke: 'black',
-      strokeWidth: 4,
+      strokeWidth: 2,
```

Comment:
i think you need to scale this against card size

Reply:
possible

Reply:
oh seems it's not necessary, since the card gets drawn as an image. For example, the finesse borders scale properly even though their width is constant